workspace/notebook/manage_main_pipeline_config_entry.json

```diff
@@ -0,0 +1,373 @@
+				"type": "Spark",
+				"endpoint": "https://pins-synw-odw-dev-uks.dev.azuresynapse.net/livyApi/versions/2019-11-01-preview/sparkPools/pinssynspodw34",
+				"auth": {
+					"type": "AAD",
+					"authResource": "https://dev.azuresynapse.net"
+				},
+				"sparkVersion": "3.4",
+				"nodeCount": 3,
+				"cores": 4,
+				"memory": 28,
+				"automaticScaleJobs": false
+			},
+			"sessionKeepAliveTimeout": 30
+		},
+		"cells": [
+			{
+				"cell_type": "markdown",
+				"metadata": {
+					"nteract": {
+						"transient": {
+							"deleting": false
+						}
+					}
+				},
+				"source": [
+					"# Notebook: manage_main_pipeline_config_entry\n",
+					"\n",
+					"# manage_main_pipeline_config\n",
+					"\n",
+					"This notebook allows controlled insertion of new entities into `main_pipeline_config` used by the ODW ingestion pipeline.\n",
+					"\n",
+					"⚠️ Do not run the notebook end to end ⚠️ Only run the cells that are needed to be run \n",
+					"\n",
+					"It provides a parameter-driven approach to:\n",
+					"- View current max `System_key` & next available key\n",
+					"- Insert new rows safely\n",
+					"- Confirm the row was added\n",
+					"- Enable/Disable the entity\n",
+					"- List all the entities\n",
+					"- Delete an entity\n",
+					"\n",
+					"⚠️ Set `is_enabled = False` ⚠️ during first insert if the corresponding Function App endpoint does not exist yet.\n",
+					"\n",
+					"\n",
+					"## Purpose\n",
+					"- Add a **new Service Bus entity** with a unique `System_key`\n",
+					"- Avoid overwriting existing entries (unlike the legacy `CREATE OR REPLACE` approach)\n",
+					"- Toggle `is_enabled` for specific entities (once their Function Apps are deployed)\n",
+					"- Validate inserts with clean, parameter-driven logic\n",
+					"\n",
+					"## How to Use\n",
+					"1. Set the parameters in the cell below.\n",
+					"2. Run the cells to insert and verify the entry.\n",
+					"3. After Function App deployment, return to update `is_enabled = true` if needed.\n",
+					""
+				]
+			},
+			{
+				"cell_type": "markdown",
+				"metadata": {
+					"nteract": {
+						"transient": {
+							"deleting": false
+						}
+					}
+				},
+				"source": [
+					"### Set Parameters"
+				]
+			},
+			{
+				"cell_type": "code",
+				"source": [
+					"# Set the details of the new entity below\n",
+					"\n",
+					"# Example:\n",
+					"# new_entity_key = 28\n",
+					"# system_name = \"Service bus\"\n",
+					"# object_name = \"Appeal Event Estimate\"\n",
+					"# entity_name = \"appeal-event-estimate\"\n",
+					"# is_enabled = False  # Set to True only after Function App is deployed\n",
+					"\n",
+					"\n",
+					"new_entity_key =   # Run next cell to get the next available key\n",
+					"system_name = \"\"\n",
+					"object_name = \"\"\n",
+					"entity_name = \"\"\n",
+					"is_enabled = False  # Use the update cell at the bottom to enable it once function app is deployed using true"
+				],
+				"execution_count": null
```

Comment:
Could this be converted to a parameter cell? (or is that overkill)

Reply:
I'd do that